.tekton/coo-fbc-v4-14-push.yaml

```diff
@@ -172,11 +191,15 @@ spec:
         values:
         - "true"
       workspaces:
-      - name: output
-        workspace: workspace
       - name: basic-auth
         workspace: git-auth
     - name: build-container
```

Comment:
this is still wrong, build-container should be renamed to build-images. I assume you meant to *remove* this line, as I see build-images later in the code.

Reply:
done